quiche/src/recovery/rtt.rs

```diff
@@ -1,94 +1,87 @@
 use core::time::Duration;
 use std::time::Instant;
 
-use crate::minmax;
+use crate::minmax::Minmax;
 
 pub(crate) const INITIAL_RTT: Duration = Duration::from_millis(333);
 
 pub(crate) const RTT_WINDOW: Duration = Duration::from_secs(300);
+pub struct RttStats {
+    pub(super) latest_rtt: Duration,
+    pub(super) min_rtt: Minmax<Duration>,
+    pub(super) smoothed_rtt: Duration,
+    pub(super) rttvar: Duration,
+    pub(super) first_rtt_sample: Option<Instant>,
```

Comment:
AFAICT the value of first_rtt_samle is never used anywhere, so maybe just needs to be a bool?